### PR TITLE
N/A Updates!

### PR DESCRIPTION
**Pull request by camc314**

### CODE REVIEW

File: .github/workflows/manual-deploy.yml [hunk 1]
```diff
@@ -48,8 +59,5 @@ jobs:
       - name: Deploy Files
         run: |
           echo "$S3_BUCKET"
-          aws s3 cp pinger.js $S3_BUCKET/pinger.js
-          aws s3 cp favicon.ico $S3_BUCKET/favicon.ico
-          aws s3 cp index.html $S3_BUCKET/index.html
-          aws s3 cp styles.css $S3_BUCKET/styles.css
-          aws s3 cp urls.json $S3_BUCKET/urls.json
\ No newline at end of file
+          aws s3 sync ./out $S3_BUCKET --recursive
```
Comment (camc314) [inline .github/workflows/manual-deploy.yml:71]: @wenjunche check me please!

Comment (wenjunche) [reply .github/workflows/manual-deploy.yml:71]: don't think ever used sync,  but seems to be much better than cp

File: next.config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+    output: "export",
+    // https://cdn.openfin.co/health/deployment/index.html
+    basePath: "/health/deployment/alpha",
```
Comment (wenjunche) [inline next.config.js]: can it use `github.event.inputs.enviroment ` somehow here, instead of hard-coding

Comment (camc314) [reply next.config.js]: I updated it - sets it dynamically noe

**Review by wenjunche — APPROVED** [review]

**Pull request merged**